<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect</class>
 <widget class="QDialog" name="connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0,150), stop:1 rgba(90, 200, 2, 150));</string>
  </property>
  <widget class="QLabel" name="labelEroor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>Connectez-vous !</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>282</width>
     <height>149</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>16</y>
      <width>256</width>
      <height>113</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color:white;
background-color:rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string>Login :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">color:white;
background-color:rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string>Mot de passe :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditLogin"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMdp"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonOK">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAnnuler">
         <property name="styleSheet">
          <string notr="true">background-color:white;</string>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
